GenderPayGap.WebUI/BackgroundJobs/ScheduledJobs/SendReminderEmailsJob.cs

```diff
@@ -68,12 +67,16 @@ private void SendReminderEmailsForSectorType(SectorTypes sector, string runId, D
                         .Where(user => !user.HasBeenAnonymised)
                         .Where(
                             user => !user.ReminderEmails
-                                .Where(re => re.SectorType == sector)
-                                .Any(re => re.DateChecked > latestReminderEmailDate));
+                                .Where(
+                                    re => re.SectorType == sector
+                                          && re.ReminderDate.HasValue
+                                          && re.ReminderDate.Value.Date == latestReminderEmailDate.Date)
```

Comment:
I didn't use functions when building the queries so that Entity Framework can translate the LINQ into SQL automatically. Otherwise, it would need to download all the entries into the memory and further parse them.